Cargo.toml

```diff
@@ -10,6 +10,156 @@ members = [
     "libs/*",
 ]
 
+[workspace.package]
+edition = "2021"
+
+## All dependency versions, used in the project, grouped in some arbitrary order.
+[workspace.dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_with = "2.0"
+
+once_cell = "1.13"
+parking_lot = "0.12"
+scopeguard = "1.1"
+rand = "0.8"
+regex = "1.4"
+itertools = "0.10"
+num-traits = "0.2.15"
+signal-hook = "0.3"
+pin-project-lite = "0.2"
+git-version = "0.3"
+const_format = "0.2"
+crc32c = "0.6"
+hashbrown = "0.13"
+uuid = { version = "1.2", features = ["v4", "serde"] }
+atty = "0.2.14"
+
+humantime = "2.1"
+humantime-serde = "1.1.1"
+chrono = { version = "0.4", default-features = false, features = ["clock"] }
+
+anyhow = { version = "1.0", features = ["backtrace"] }
+thiserror = "1.0"
+
+futures-core = "0.3"
+futures-util = "0.3"
+futures = "0.3"
+async-stream = "0.3"
+async-trait = "0.1"
+tokio = { version = "1.17", features = ["macros"] }
+tokio-postgres-rustls = "0.9.0"
+tokio-util = { version = "0.7", features = ["io"] }
+tokio-stream = "0.1"
+tokio-rustls = "0.23"
+
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+
+toml = "0.5"
+toml_edit = { version = "0.17", features = ["easy"] }
+base64 = "0.13.0"
+md5 = "0.7.0"
+svg_fmt = "0.4.1"
+hex = "0.4"
+hex-literal = "0.3"
+
+clap = "4.0"
+
+tar = "0.4"
+rstar = "0.9.3"
+
+rustls = "0.20"
+rustls-split = "0.3"
+rustls-pemfile = "1"
+
+url = "2.2"
+serde_json = "1"
+routerify = "3"
+jsonwebtoken = "8"
+tls-listener = { version = "0.6", features = ["rustls", "hyper-h1"] }
+
+reqwest = { version = "0.11", default-features = false, features = ["rustls-tls"] }
+hyper = "0.14"
+socket2 = "0.4.4"
+hyper-tungstenite = "0.9"
+tonic = {version = "0.8", features = ["tls", "tls-roots"]}
+prost = "0.11"
+
+aws-smithy-http = "0.51.0"
+aws-types = "0.51.0"
+aws-config = { version = "0.51.0", default-features = false, features=["rustls"] }
+aws-sdk-s3 = "0.21.0"
+
+sentry = { version = "0.29", default-features = false, features = ["backtrace", "contexts", "panic", "rustls", "reqwest" ] }
+prometheus = {version = "0.13", default_features=false, features = ["process"]} # removes protobuf dependency
+
+bindgen = "0.61"
+libc = "0.2"
+memoffset = "0.8"
+bincode = "1.3"
+nix = "0.26"
+bytes = "1.0"
+byteorder = "1.4"
+
+strum = "0.24"
+strum_macros = "0.24"
+notify = "5.0.0"
+fs2 = "0.4.3"
+comfy-table = "6.1"
+hmac = "0.12.1"
+bstr = "1.0"
+close_fds = "0.3.2"
+fail = "0.5.0"
+crossbeam-utils = "0.8.5"
+sha2 = "0.10.2"
+webpki-roots = "0.22.5"
+x509-parser = "0.14"
+walkdir = "2.3.2"
+
+## TODO replace this with tracing
+log = "0.4"
+env_logger = "0.10"
+
+# TODO fork this and use similar `rev` hash 
+amplify_num = { git = "https://github.com/hlinnaka/rust-amplify.git", branch = "unsigned-int-perf" }
```

Comment:
I guess we can switch back, because @hlinnaka’s PR was merged

Reply:
I see that this PR seems to be included into `4.0.0-*` branch: https://github.com/rust-amplify/rust-amplify/commit/bd49b737c2e6e623ab8e9ba5ceaed5712d3a3940

but that's still not released.
Switched to the upstream git repo at least.